### PR TITLE
Add fmf_path

### PR DESCRIPTION
**Pull request by LecrisUT**
TODO:

- [x] Write new tests or update the old ones to cover new functionality.
- [x] Update doc-strings where appropriate.
- [x] Update or write new documentation in `packit/packit.dev`.

Fixes https://github.com/packit/packit/issues/1935

Merge after [packit#1936](https://github.com/packit/packit/pull/1936)

---

RELEASE NOTES BEGIN
Allow running tmt tests with fmf root outside of git root
RELEASE NOTES END

### CODE REVIEW

File: packit_service/worker/helpers/testing_farm.py [hunk 1]
```diff
@@ -295,7 +301,10 @@ def _payload_without_token(payload: Dict) -> Dict:
         return payload_
 
     def _construct_fmf_payload(self) -> dict:
-        fmf = {"url": self.fmf_url}
+        fmf = {
+            "url": self.fmf_url,
+            "path": self.fmf_path,
+        }
```
Comment (lachmanfrantisek) [inline packit_service/worker/helpers/testing_farm.py:307]: What about having it optional and not sending anything if not configured? (Similar to the fmf_ref option.)

Comment (lachmanfrantisek) [reply packit_service/worker/helpers/testing_farm.py:307]: (But I see it's needed to check if the FMF is configured.)

**Review by lachmanfrantisek — APPROVED** [review]
Nice, Cristian! Thanks!

File: packit_service/worker/helpers/testing_farm.py (hunk 1 above)
Comment (LecrisUT) [reply packit_service/worker/helpers/testing_farm.py:307]: So far this has worked without it on testing-farm side, altgough the api appears to be outdated either on this side or theirs (`tmt` vs `fmf` naming).

We are doing a fmf testing on packit side, so might be better for debugging down the line to be explicit.

**Comment by LecrisUT** [comment]
Recheck

**Comment by LecrisUT** [comment]
recheck

**Comment by lbarcziova** [comment]
recheck

**Comment by LecrisUT** [comment]
I am a bit confused about `flexmock`, shouldn't this be defined on the `koji`, `testing-farm`, etc. so that they guarantee the appropriate API version?

**Comment by LecrisUT** [comment]
Hmm latest push is not updating this PR?

**Comment by LecrisUT** [comment]
recheck

**Comment by LecrisUT** [comment]
Requesting review about: 6b37933e6eb107005b9a5b229ad17fbe81b49493

**Review by lachmanfrantisek — APPROVED** [review]
Thanks @LecrisUT !

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
recheck

**Pull request merged**